### PR TITLE
#232 Added logging when exploring libraries (jars or folders)

### PR DESCRIPTION
**Pull request by jperedadnr**
including refactoring of JarExplorer and FolderExplorer
#233 Don’t scan JavaFX jars

This PR adds some really useful information to the console when importing custom libraries, mainly focused on library developers that never know why Scene Builder doesn't want to import their controls.

The logging level is set to `info`, but in most cases, a regular user will launch Scene Builder from the executable, without a visible console.

### CODE REVIEW

**Comment by mimmoz81** [comment]
That seems good to me.
ExplorerBase was exactly what I was mentioning in my comments at #228

File: kit/src/main/java/com/oracle/javafx/scenebuilder/kit/library/BuiltinLibrary.java
Comment (abhinayagarwal) [inline kit/src/main/java/com/oracle/javafx/scenebuilder/kit/library/BuiltinLibrary.java]: Move this method up and place it along with other public methods. If possible, add some Javadoc.

File: kit/src/main/resources/com/oracle/javafx/scenebuilder/kit/i18n/SceneBuilderKit.properties
Comment (abhinayagarwal) [inline kit/src/main/resources/com/oracle/javafx/scenebuilder/kit/i18n/SceneBuilderKit.properties]: This property is not in use and can be removed.

File: kit/src/main/java/com/oracle/javafx/scenebuilder/kit/library/util/JarExplorer.java
Comment (abhinayagarwal) [inline kit/src/main/java/com/oracle/javafx/scenebuilder/kit/library/util/JarExplorer.java]: This would be a good opportunity to add some Javadoc to the classes explaining what these classes do.

**Comment by nathanwonnacott** [comment]
Given that a year and a half has passed, it seems to me like @jperedadnr isn't likely to address the comments on the merge request. Would it be possible to just merge this and then someone with write access to the gluonhq repo could make the suggested changes after the merge? The suggested changes are quite small.

File: kit/src/main/java/com/oracle/javafx/scenebuilder/kit/library/util/JarExplorer.java
Comment (jperedadnr) [reply kit/src/main/java/com/oracle/javafx/scenebuilder/kit/library/util/JarExplorer.java:46]: This will require some time, probably best for a folluw-up.

File: kit/src/main/resources/com/oracle/javafx/scenebuilder/kit/i18n/SceneBuilderKit.properties
Comment (jperedadnr) [reply kit/src/main/resources/com/oracle/javafx/scenebuilder/kit/i18n/SceneBuilderKit.properties:75]: Fixed

File: kit/src/main/java/com/oracle/javafx/scenebuilder/kit/library/BuiltinLibrary.java
Comment (jperedadnr) [reply kit/src/main/java/com/oracle/javafx/scenebuilder/kit/library/BuiltinLibrary.java]: Done

**Comment by jperedadnr** [comment]
@nathanwonnacott Sorry, it took some time... but I addressed the comments. We'll merge soon.

File: kit/src/main/java/com/oracle/javafx/scenebuilder/kit/library/util/FolderExplorer.java
Comment (abhinayagarwal) [inline kit/src/main/java/com/oracle/javafx/scenebuilder/kit/library/util/FolderExplorer.java:2]: Shouldn't this change be "2017, 2020, Gluon" ?

**Review by abhinayagarwal — APPROVED** [review]

File: kit/src/main/java/com/oracle/javafx/scenebuilder/kit/library/util/FolderExplorer.java
Comment (jperedadnr) [reply kit/src/main/java/com/oracle/javafx/scenebuilder/kit/library/util/FolderExplorer.java:2]: No, that file was actually created in 2019, in this PR https://github.com/gluonhq/scenebuilder/pull/228

**Pull request merged**
